packages/react-native-renderer/src/ReactFabricEventResponderSystem.js

```diff
@@ -106,7 +106,6 @@ const eventResponderContext: ReactNativeResponderContext = {
       }
       case ContinuousEvent: {
         executeUserEventHandler(eventListener, eventValue);
-        break;
```

Comment:
We should probably leave the `break` in here and in the DOM version in case any new cases get added later